src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -251,161 +253,223 @@ public InventoryType getType()
 	@Override
 	public int getMaxStackSize()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return 64;
 	}
 	
 	@Override
-	public void setMaxStackSize(int size)
-	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
-		
-	}
+	public void setMaxStackSize(int size) {}
 	
```

Comment:
I've meant InventoryMock.setMaxStackSize(int) method. This method sets a limit of how many items can be in a stack, in a certain inventory, but there's no implementation of that (doesn't need to be implemented, but it should have this old UnimplementedOperationException throw and TODO comment instead

Reply:
done